### PR TITLE
fix: suppress buggy output from pytensor.function

### PR DESCRIPTION
**Pull request by cpaniaguam**
See #594.

### CODE REVIEW

**Review by AlexanderFengler — COMMENTED** [review]
Is this functionality meant to sit in `test_likelihoods.py` or is this wip?

**Comment by cpaniaguam** [comment]
`pytensor.function` is also used in https://github.com/lnccbrown/HSSM/blob/fa12018b146596505b403a4bdf2d7fb9ea475570/src/hssm/distribution_utils/dist.py#L413

It could be used there as well right now.

@AlexanderFengler Where would a good home be for the output suppressor?

**Review by digicosmos86 — APPROVED** [review]
Looks good to me! Just one comment: we use numpy docstring format. The docstring is slightly non-standard :)

File: src/hssm/utils.py [hunk 1]
```diff
@@ -548,3 +550,43 @@ def _rearrange_data(data: pd.DataFrame | np.ndarray) -> pd.DataFrame | np.ndarra
 def _split_array(data: np.ndarray | list[int], divisor: int) -> list[np.ndarray]:
     num_splits = len(data) // divisor + (1 if len(data) % divisor != 0 else 0)
     return [tmp.astype(int) for tmp in np.array_split(data, num_splits)]
+
+
+class SuppressOutput:
```
Comment (AlexanderFengler) [inline src/hssm/utils.py:555]: sorry about the late response here, but his converged to the right spot :)

**Review by AlexanderFengler — APPROVED** [review]
lgtm

**Pull request merged**